src/udev/probes.d

```diff
@@ -0,0 +1,11 @@
+provider udev {
```

Comment:
hmm, they use ".d" as suffix for a file? how unfortunate since that's usually used for drop-in style dirs

Reply:
They used that suffix waaay before we came up with idea of drop-in directories.